libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -161,6 +161,40 @@ const ConnectorList = ({
     <>
       <ConnectDialogContent>
         <ConnectDialogTitle>{t('Connect')}</ConnectDialogTitle>
+        {urlInputExpanded ? (
```

Comment:
suggestion(non-blocking): Perhaps best to refactor these into separate components and make them reusable. Otherwise abstract to top as consts.